### ISSUE TITLE
Github release is no longer updating

### ISSUE THREAD

**Issue by CaledoniaProject** [x64dbg/x64dbg#2715]
Hi,

The last snapshot is more than 1 month old, have you disabled the automated nightly build?
https://github.com/x64dbg/x64dbg/releases

**Comment by mrexodia** [comment]
Yes, my CI machine died and I’m working on recovering it. In the meantime
you can still get the latest builds from AppVeyor (green checkmark next to
the commit)

**Comment by CaledoniaProject** [comment]
Thanks, consider using github workflow to upload the snapshot.

**Comment by justanotheranonymoususer** [comment]
@mrexodia change http://snapshots.x64dbg.com/ to point to AppVeyor? I didn't know the files are stalled.
